src/com/owncloud/android/ui/fragment/LocalFileListFragment.java

```diff
@@ -98,6 +98,18 @@ public void onActivityCreated(Bundle savedInstanceState) {
         
         Log_OC.i(TAG, "onActivityCreated() stop");
     }
+
+    public void CheckAllFile(boolean value)
```

Comment:
checkAllFiles <- lowercase at first char & plural

check is more of UI style naming, maybe selectAllFiles